core/extstate/database_test.go

```diff
@@ -352,7 +352,7 @@ func FuzzTree(f *testing.F) {
 					fuzzState.deleteStorage(rand.Intn(len(fuzzState.currentAddrs)), rand.Uint64())
 				}
 			default:
-				t.Fatalf("unknown step: %d", step)
+				require.Failf(t, "unknown step", "got: %d", step)
```

Comment:
This is an overreaching linter that replaces stdlib features with something that does nothing more than require the user to know that it's the same thing.